demo/site/src/documents/pages/blocks/ContentGroupBlock.tsx

```diff
@@ -37,24 +39,9 @@ const ContentGroupContentBlock = withPreview(
 
 export const ContentGroupBlock = withPreview(
     ({ data: { content, backgroundColor } }: PropsWithData<ContentGroupBlockData>) => (
-        <RootPageLayout $background={backgroundColor}>
+        <PageLayout className={clsx(backgroundColor === "lightGray" && styles.lightGray, backgroundColor === "darkGray" && styles.darkGray)}>
```

Comment:
This would work as well:

```suggestion
        <PageLayout className={styles[backgroundColor]}>
```

Reply:
yep, i thought about that too. 
I decided for the "type safer" option, but we used this variant on other places before so i guess its ok :)